docs/data/routes/docs/nextjs/tracking-and-analytics/overview/en.md

```diff
@@ -3,4 +3,40 @@ name: overview
 routeTemplate: ./data/component-templates/article.yml
 title: Overview and architecture
 ---
-# Overview and architecture
\ No newline at end of file
+# Overview and architecture
```

Comment:
I would change the title to "Tracking & Analytics Overview and Architecture"

Reply:
Updated.